strr-examiner-web/app/components/SupportingDocuments.vue

```diff
@@ -52,8 +52,9 @@ const appRegNumber = computed((): string | number =>
         {{ t(`documentLabels.${document.documentType}`) }}
       </UButton>
       <UBadge
-        v-if="document.uploadStep && props.config?.includeDateBadge?.includes(document.uploadStep)"
-        :label="`${ t('strr.label.added')} ` + document.uploadDate"
+        v-if="(document.uploadStep && props.config?.includeDateBadge?.includes(document.uploadStep)) ||
```

Comment:
Maybe this could be extracted to a prop - it's quite long condition.

Reply:
Done